### ISSUE TITLE
unstable training

### ISSUE THREAD

**Issue by Feynman1999** [The-Learning-And-Vision-Atelier-LAVA/LLT#3]
I am training a super-resolution network using your method, but I have noticed that even after loading a pre-trained model, the PSNR during validation suddenly drops significantly after a few hundred or a few thousand iterations (for example, from 38 to 32). The reason seems to be that the loss for one iteration suddenly increases. Have you encountered this situation before? What could be the possible reasons for it? Thank you!"
